sql/bgw_scheduler.sql

```diff
@@ -1,16 +1,19 @@
 CREATE OR REPLACE FUNCTION _timescaledb_internal.restart_background_workers()
-RETURNS BOOL 
+RETURNS BOOL
 AS '@LOADER_PATHNAME@', 'ts_bgw_db_workers_restart'
 LANGUAGE C VOLATILE;
 
 CREATE OR REPLACE FUNCTION _timescaledb_internal.stop_background_workers()
-RETURNS BOOL 
+RETURNS BOOL
 AS '@LOADER_PATHNAME@', 'ts_bgw_db_workers_stop'
 LANGUAGE C VOLATILE;
 
 CREATE OR REPLACE FUNCTION _timescaledb_internal.start_background_workers()
-RETURNS BOOL 
+RETURNS BOOL
 AS '@LOADER_PATHNAME@', 'ts_bgw_db_workers_start'
 LANGUAGE C VOLATILE;
 
 SELECT _timescaledb_internal.start_background_workers();
+
+INSERT INTO _timescaledb_config.bgw_job (application_name, job_type, schedule_INTERVAL, max_runtime, max_retries, retry_period) VALUES
```

Comment:
Can I (or you) add an `ON CONFLICT DO NOTHING` here? The update CI tests are failing here with a . I've tried removing it from the update modfile but it still doesn't work, so I'm not sure where it's originally getting inserted but an `ON CONFLICT` seems safe?